src/test/system_tests/ceph_s3_tests/test_ceph_s3.js

```diff
@@ -76,8 +75,9 @@ async function run_s3_tests() {
 
 async function run_all_tests() {
     console.info('Running Ceph S3 Tests...');
+    const tox_args = `-c ${CEPH_TEST.test_dir}${CEPH_TEST.s3_test_dir}${CEPH_TEST.tox_config}`;
     const tests_list_command =
-        `S3TEST_CONF=${CEPH_TEST.test_dir}${CEPH_TEST.ceph_config}  ./${CEPH_TEST.test_dir}${CEPH_TEST.s3_test_dir}virtualenv/bin/nosetests -v --collect-only  2>&1 | awk '{print $1}' | grep test`;
+        `S3TEST_CONF=${process.cwd()}/${CEPH_TEST.test_dir}${CEPH_TEST.ceph_config} tox ${tox_args} -- -q --collect-only --disable-pytest-warnings  2>&1 | awk '{print $1}' | grep test`;
```

Comment:
Why did you add `process.cwd()`?

Reply:
its instead of $PWD. tox doesn't recognize the relative path, so I use the full path